src/__test__/components/data-management/AnalysisModal.test.jsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import { List } from 'antd';
+import { configure, mount } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+import { Provider } from 'react-redux';
+import thunk from 'redux-thunk';
+import '@testing-library/jest-dom';
+import configureMockStore from 'redux-mock-store';
+import AnalysisModal from '../../../components/data-management/AnalysisModal';
+
+configure({ adapter: new Adapter() });
+
+const mockStore = configureMockStore([thunk]);
+
+const initialState = {
+  projects: {
+    meta: {
+      activeProject: 'mock-project',
+    },
+  },
+  'mock-project': {
+    name: 'Mock project',
+    experiments: ['mock-experiment'],
+  },
+  experiments: {
+    ids: ['mock-experiment'],
+    'mock-experiment': {
+      name: 'Mock Experiment',
+      id: 'mock-experiment',
+    },
+  },
+};
+
+const activeProject = initialState['mock-project'];
+const { experiments } = initialState;
+
+describe('AnalysisModal', () => {
```

Comment:
we need a unit test to check if renaming works -- rename of the experiment name and changing of the description

Reply:
written